impl/src/test/groovy/com/okta/sdk/impl/resource/DefaultUserBuilderTest.groovy

```diff
@@ -212,6 +212,43 @@ class DefaultUserBuilderTest {
                     aMapWithSize(4))
     }
 
+    @Test
+    void importPasswordMd5() {
```

Comment:
I'd like to see a test here that does the e2e of 
```
create user
import md5 password
log the user in, with the original password that was used during password import
```

Reply:
Actually, it would be sufficient to test the creation of user with MD5 password import in the scope of this Mgmt SDK. Testing the login functionality of the created user falls in the realm of the Authn SDK and we can skip that part IMHO.

Reply:
I have to disagree here. We need to make sure that the import works from within the management SDK.  It doesn't hurt to have that assurance here, as well as in the Authn SDK

Reply:
@VitaliiTytarenko-okta let us add login test here, but DO NOT add Authn SDK as a dependency. We should just be able to use a simple REST client to access the Authn API to accomplish the login functionality. Connect with me for more details.